### ISSUE TITLE
make push-docker does not tag release as latest

### ISSUE THREAD

**Issue by sanand0** [gramener/gramex#320]
`make push-docker` does not update the `gramener/gramex:latest` tag. It should, I think.

Adding this line seems to work;

```python
client.api.tag(f'gramener/gramex:{release["version"]}', 'gramener/gramex', 'latest')
```

Shall I go ahead @jaidevd?

**Comment by jaidevd** [comment]
@sanand0 I think "latest" should not be a tag at all. The image should be tagged with the latest gramex version - corresponding to the git tag / PyPI release. When doing a `docker pull` (without specifying a tag), docker should automatically pull the latest tag, release-wise. We should just change the last argument here to `f{release['version']}`

**Comment by sanand0** [comment]
Closed via c5b3a133
